Gemfile

```diff
@@ -56,6 +56,7 @@ gem 'rack-cors', require: 'rack/cors'
 gem 'rainbow'
 
 # Use ActiveModel has_secure_password
+gem 'nokogiri', '>= 1.13.2'
```

Comment:
It doesn't look like this PR needs Nokogiri. Is there a reason for this addition? Normally the Gemfile won't include dependencies not directly needed.

Reply:
I had it in from the previous branch when I was running into an issue before I added the version to the .ruby_version file, its removed now